diff --git a/node_modules/react-native/scripts/ios-configure-glog.sh b/node_modules/react-native/scripts/ios-configure-glog.sh
index dd79583..905fb8f 100755
--- a/node_modules/react-native/scripts/ios-configure-glog.sh
+++ b/node_modules/react-native/scripts/ios-configure-glog.sh
@@ -20,6 +20,27 @@ if [ -z "$CURRENT_ARCH" ] || [ "$CURRENT_ARCH" == "undefined_arch" ]; then
     fi
 fi

+if [ "$CURRENT_ARCH" == "arm64" ]; then
+    cat <<\EOF >>fix_glog_0.3.5_apple_silicon.patch
+diff --git a/config.sub b/config.sub
+index 1761d8b..43fa2e8 100755
+--- a/config.sub
++++ b/config.sub
+@@ -1096,6 +1096,9 @@ case $basic_machine in
+        basic_machine=z8k-unknown
+        os=-sim
+        ;;
++   arm64-*)
++       basic_machine=$(echo $basic_machine | sed 's/arm64/aarch64/')
++       ;;
+    none)
+        basic_machine=none-none
+        os=-none
+EOF
+    pwd
+    patch -p1 --ignore-whitespace --forward config.sub fix_glog_0.3.5_apple_silicon.patch
+fi
+
 export CC="$(xcrun -find -sdk $PLATFORM_NAME cc) -arch $CURRENT_ARCH -isysroot $(xcrun -sdk $PLATFORM_NAME --show-sdk-path)"
 export CXX="$CC"

diff --git a/node_modules/react-native/scripts/react_native_pods.rb b/node_modules/react-native/scripts/react_native_pods.rb
index e55b6f5..da6adbf 100644
--- a/node_modules/react-native/scripts/react_native_pods.rb
+++ b/node_modules/react-native/scripts/react_native_pods.rb
@@ -76,7 +76,7 @@ def use_flipper!(versions = {}, configurations: ['Debug'])
   versions['Flipper-DoubleConversion'] ||= '3.1.7'
   versions['Flipper-Fmt'] ||= '7.1.7'
   versions['Flipper-Folly'] ||= '2.6.7'
-  versions['Flipper-Glog'] ||= '0.3.6'
+  versions['Flipper-Glog'] ||= '0.3.9'
   versions['Flipper-PeerTalk'] ||= '0.0.4'
   versions['Flipper-RSocket'] ||= '1.4.3'
   pod 'FlipperKit', versions['Flipper'], :configurations => configurations
